cmd/pkgversion/pkgversion.go

```diff
@@ -15,19 +15,25 @@ import (
 
 func CheckVersion() *cobra.Command {
 
-	var owner string
-	var repo string
+	var repoSlug string
 	cmd := &cobra.Command{
 		Use:   "pkgversion",
 		Short: "Check a Pulumi package version",
 		Long:  `Get the most recent version of a Pulumi package and compare with the version in the registry`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			latest := fmt.Sprintf("https://api.github.com/repos/%s/%s/releases/latest", owner, repo)
+			latest := fmt.Sprintf("https://api.github.com/repos/%s/releases/latest", repoSlug)
 			version, err := getLatestVersion(latest)
 			if err != nil {
 				return err
 			}
-			pkgName := strings.TrimPrefix(repo, "pulumi-")
+
+			repoName := ""
+			githubSlugParts := strings.Split(repoSlug, "/")
+			if len(githubSlugParts) > 0 {
+				repoName = githubSlugParts[1]
+			}
```

Comment:
same, really - we want to at least error out if `githubSlugParts` is of length greater than 2